### PR TITLE
Edits to Acme Telco Lite tutorial

### PR DESCRIPTION
**Pull request by alexronquillo**
- Updated content to match updated deployment
- Updated images
- Added second WIP story placeholder

### CODE REVIEW

File: documentation/tutorial/user_stories/AcmeTelcoLite/README.md [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+* Inventory Service
+* Plan Service
+* Fulfillment Service
+* Billing Service
+
+The simulator is not part of the Telco Lite infrastructure, but it is part of the demo deployment. It runs scenarios against the application to create web traffic and generate interesting data in New Relic.
+
+## Architecture
+
+![Acme Telco Lite architecture diagram](imgs/acme.png)
```
Comment (Julien4218) [inline documentation/tutorial/user_stories/AcmeTelcoLite/README.md:22]: That diagram is old and outdated, we need a new one

**Comment by Julien4218** [comment]
This is quite nice @alexronquillo I think we could merge that in, although the architecture diagram is old and inaccurate and would need to be replaced.
There is another piece that could be added to the CPU storyline, after establishing the cause of the problem, we could take the users to navigate to the Web Portal application (from the python Fulfillment Service dependency) and see that this also impacts the other users on a different application (not only the Warehouse portal).
Just a suggestion, feel free to run with it or ask around.
Let me know if you'd like a new diagram before merging this work.

**Comment by alexronquillo** [comment]
@Julien4218 Thank you, an accurate diagram would be great! I'll extend the tutorial one more segment to show effects on the Web Portal, like you mentioned.

**Comment by alexronquillo** [comment]
@Julien4218 I added a section on using the service map to identify the web portal's dependency on the fulfillment service.

**Pull request merged**